doc/content/reference/application-packages/using-the-api.md

```diff
@@ -0,0 +1,262 @@
+```
+
+This gives the result in the JSON format:
+
+```json
+{
+  "packages": [
+    {
+      "name": "aws-iot",
+      "default_f_port": 198
+    },
+    {
+      "name": "azure-iot-hub",
+      "default_f_port": 196
+    },
+    {
+      "name": "lora-cloud-geolocation-v3",
+      "default_f_port": 197
+    },
+    {
+      "name": "storage-integration",
+      "default_f_port": 200
+    },
+    {
+      "name": "lora-cloud-device-management-v1",
+      "default_f_port": 199
+    }
+  ]
+}
+```
+
+In the following examples, replace the `test-package` with any of the packages listed above.
+
+## Creating Associations
+
+To create a default association between the application package and the FPort `100` of an application `app1`:
+
+```bash
+curl 'https://eu1.cloud.thethings.network/applications/app1/packages/associations/100' \
+  -X 'PUT' \
+  -H 'content-type: application/json' \
+  -H 'authorization: Bearer NNSXS.XXXXXXXXX' \
+  --data-raw '{"default":{"package_name":"test-package","field_mask":{"paths":["package_name"]}}}'
+```
+
+The default association will associate the application package with all devices within a given application.
+
+<details><summary>Output</summary>
+
+```json
+{
+  "ids": {
+    "application_ids": {
+      "application_id": "app1"
+    },
+    "f_port": 100
+  },
+  "created_at": "2021-12-30T10:26:31.302076317Z",
+  "updated_at": "2021-12-30T10:26:31.302076317Z"
+}
+```
+</details>
+
+To create an association between the application package and the FPort `100` of a device `dev1` of an application `app1`:
+
+```bash
+curl 'https://eu1.cloud.thethings.network/applications/app1/devices/dev1/packages/associations/100' \
+  -X 'PUT' \
+  -H 'content-type: application/json' \
+  -H 'authorization: Bearer XXXXXXXXX' \
+  --data-raw '{"association":{"package_name":"test-package","field_mask":{"paths":["package_name"]}}}'
+```
+
+This will associate the application package with a single device within a given application.
+
+<details><summary>Output</summary>
+
+```json
+{
+  "ids": {
+    "end_device_ids": {
+      "device_id": "dev1",
+      "application_ids": {
+        "application_id": "app1"
+      }
+    },
+    "f_port": 100
+  },
+  "created_at": "2021-12-30T10:55:09.496854413Z",
+  "updated_at": "2021-12-30T10:55:09.496854413Z"
+}
+```
+</details>
+
+## Listing Associations
+
+To list all default associations associated with an application `app1`:
+
+```bash
+curl 'https://eu1.cloud.thethings.network/api/v3/as/applications/app1/packages/associations' \
+  -X 'GET' \
+  -H 'content-type: application/json' \
+  -H 'authorization: Bearer XXXXXXXXX'
+```
+
+<details><summary>Output</summary>
+
+```json
+{
+  "defaults": [
+    {
+      "ids": {
+        "application_ids": {
+          "application_id": "app1"
+        },
+        "f_port": 100
+      },
+      "created_at": "2021-12-30T10:26:31.302076317Z",
+      "updated_at": "2021-12-30T10:26:31.302076317Z",
+      "package_name": "test-package"
```

Comment:
@adriansmares I'm curious about this one. When I list associations, I don't get the `package_name` for any association other than `lora-cloud-device-management-v2` (tried `storage-integration` and `lora-cloud-geolocation-v3` on default ports). I added the `package_name` entry here manually just to point out what's confusing me, but in the original output there is no `package_name` in this place. If there is many associations, how can one differ one association from another from this list if all you can see is the application ID, port, and the time when the association was created/updated?